index.html

```diff
@@ -4189,7 +4189,22 @@ <h3>JSON-LD</h3>
         </p>
 
         <p>
-[[!JSON-LD11]] is useful when extending the data model described in this
+Linked Data is modeled using Resource Description Framework (RDF).
+RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity→attribute→value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD [[?JSON-LD11]] is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications that consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if necessary, using the <code>id</code> key in this
+specification. See
+<a data-cite="?JSON-LD11#relationship-to-rdf">Relationship of JSON-LD to RDF</a>
+for more details.
+          </p>
+        <p>
+Consequently, [[!JSON-LD11]] is useful when extending the data model described in this
```

Comment:
```suggestion
[[!JSON-LD11]] is useful when extending the data model described in this
```

"Consequently" to what? I think it is unnecessary here.

Reply:
JSON-LD provides all the super-powers described in the preceding paragraph...consequently, it's "useful when extending the data model described in this specification."

If that's not generally clear, though, I can forego the "consequently." 😄

Reply:
But this is certainly not a reason to lie down on the road as far as I am concerned... 😀

Reply:
I agree with @iherman, let's take it out, it read a bit strange to me as well. I get what it's saying, but removing it didn't harm the text, IMHO.